core/src/main/java/org/frankframework/util/MessageUtils.java

```diff
@@ -332,9 +333,15 @@ private static MimeType guessMimeType(Message message) {
 		if (!"{".equals(firstChar) && !"[".equals(firstChar)) {
 			return MediaType.TEXT_PLAIN;
 		}
-		try (JsonParser parser = Json.createParser(message.asInputStream())) {
-			parser.next();
-			return MediaType.APPLICATION_JSON;
+		try {
+			InputStream inputStream = message.asInputStream();
+			inputStream.mark(20_000);
```

Comment:
Ik weet niet of de jsonParser meer wilt inlezen en dit nu goed gaat omdat onze test input niet groter is dan 20_000L

Reply:
PS: De parser is een streaming parser. Hij zal niet meteen de hele JSON inlezen, behalve misschien in bepaalde edge-cases.
Daarom verwacht ik niet direct een probleem.

Reply:
Ik heb 2 tests toegevoegd voor grote JSON berichten. Lijkt te werken, in ieder geval met deze berichten.